src/bmp/BMPParser.h

```diff
@@ -10,10 +10,9 @@
 #include <string>
 #include <cassert>

```

Comment:
Is `iostream` used?

Looks like the `cassert` include can be moved to the .cc file.

(`string` should be in both, since it's used in both.)